### ISSUE TITLE
Kernel Headers Could Not Be Found

### ISSUE THREAD

**Issue by versipellis** [jakeday/linux-surface#63]
Got to the last step of the install, and it threw this error, but proceeded to finish the rest of the install:

> run-parts: executing /etc/kernel/postinst.d/apt-auto-removal 4.14.5-jakeday-surface /boot/vmlinuz-4.14.5-jakeday-surface
run-parts: executing /etc/kernel/postinst.d/dkms 4.14.5-jakeday-surface /boot/vmlinuz-4.14.5-jakeday-surface
Error! Your kernel headers for kernel 4.14.5-jakeday-surface cannot be found.
Please install the linux-headers-4.14.5-jakeday-surface package,
or use the --kernelsourcedir option to tell DKMS where it's located

**Comment by jakeday** [comment]
@versipellis Did you install the headers package deb before the image package deb?

**Comment by versipellis** [comment]
I did. Actually, even the headers package deb is throwing that error.

**Comment by versipellis** [comment]
@jakeday Here's the output from trying to install the headers package deb:

> sudo dpkg -i linux-headers-4.14.5-jakeday-surface_4.14.5-jakeday-surface-10.00.Custom_amd64.deb
(Reading database ... 196599 files and directories currently installed.)
Preparing to unpack linux-headers-4.14.5-jakeday-surface_4.14.5-jakeday-surface-10.00.Custom_amd64.deb ...
Unpacking linux-headers-4.14.5-jakeday-surface (4.14.5-jakeday-surface-10.00.Custom) over (4.14.5-jakeday-surface-10.00.Custom) ...
Setting up linux-headers-4.14.5-jakeday-surface (4.14.5-jakeday-surface-10.00.Custom) ...
Examining /etc/kernel/header_postinst.d.
run-parts: executing /etc/kernel/header_postinst.d/dkms 4.14.5-jakeday-surface /boot/vmlinuz-4.14.5-jakeday-surface
Error! Your kernel headers for kernel 4.14.5-jakeday-surface cannot be found.
Please install the linux-headers-4.14.5-jakeday-surface package,
or use the --kernelsourcedir option to tell DKMS where it's located

**Comment by jakeday** [comment]
@versipellis Can you try the latest 4.14.7 build? This should hopefully be fixed.

**Comment by versipellis** [comment]
Sure thing. Should I purge the existing install first or just install the
new ones over?

**Comment by jakeday** [comment]
You can just install the new ones.

**Comment by jakeday** [comment]
@versipellis Looks like this is fixed in 4.14.7. Can you confirm?

**Comment by versipellis** [comment]
Testing it right now!

**Comment by versipellis** [comment]
Tested it, didn't work the first time round (got stuck on something else during the install), reinstalled Ubuntu and it worked this time round.

**Comment by jakeday** [comment]
Good deal!

**Comment by Predatux** [comment]
I have a related problem. The system works perfectly for me. I installed virtualbox to run Genymotion, but when I started genymotion, it tells me that virtualbox is not installed. Virtualbox is currently installed and running, but I get an error during installation that the kernel sources are not installed. I reinstalled the two files without problems....
linux-headers-4.14.7-surface-jakeday_4.14.7-surface-jakeday-9_amd64. deb
linux-image-4.14.7-surface-jakeday_4.14.7-surface-jakeday-9_amd64. deb
You know what the problem might be?
